generators/requirefilegen/src/require_file_generator.rb

```diff
@@ -0,0 +1,232 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require 'erb'
+require 'json'
+require_relative 'require_file_generator_options_parser'
```

Comment:
please use either `require` or `require_relative`. Choose one to be consistent

Reply:
This is just the generator and not the generated code